webpackjs/components/Calcs/CalcComponent.js

```diff
@@ -0,0 +1,155 @@
+                    <div>
+                        <div className="form-group">
+                            <label htmlFor="width" className="calc-label-top">
+                                Width (inches):
+                            </label>
+                            <br></br>
+                            <div className="row">
+                                <div className="col-md-8">
+                                    <input
+                                        type="range"
+                                        max={objectProps.maxWidth}
+                                        value={inputWidth}
+                                        className="slider"
+                                        onChange={getOutputWidth}
+                                    >
+                                    </input>
+                                </div>
+                                <div className="col-md-4">
+                                    <input
+                                        type="number"
+                                        max={objectProps.maxWidth}
+                                        value={inputWidth}
+                                        className="form-control"
+                                        onChange={getOutputWidth}
+                                    >
+                                    </input>
+                                </div>
+                            </div>
+                        </div>
+                        <div className="form-group">
+                            <label htmlFor="linearFeet">
+                                Linear feet:
+                            </label>
+                            <br></br>
+                            <div className="row">
+                                <div className="col-md-8">
+                                    <input
+                                        type="range"
+                                        max={objectProps.maxFeet}
+                                        value={inputLinearFeet}
+                                        className="slider"
+                                        onChange={getOutputLinearFeet}
+                                    >
+                                    </input>
+                                </div>
+                                <div className="col-md-4">
+                                    <input
+                                        type="number"
+                                        max={objectProps.maxFeet}
+                                        value={inputLinearFeet}
+                                        className="form-control"
+                                        onChange={getOutputLinearFeet}
+                                    >
+                                    </input>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    ) : (
```

Comment:
when does this empty state occur? Maybe using this syntax would be easier than using the ternary syntax: 
```js
boolean && (
   <div>
      ...
    </div>
);
```

Reply:
This is so much cleaner! Done.